src/components/YourCompany/BusinessContactInfo.vue

```diff
@@ -247,6 +240,13 @@ export default class BusinessContactInfo extends Mixins(CommonMixin) {
     return this.businessContact.email !== this.originalBusinessContact.email
   }
 
+  // Watchers
+  @Watch('businessContact', { immediate: true })
```

Comment:
If you don't watch-deep then this method will not be called if a sub-property changes; it will only be called if the main handle changes.

Reply:
Interesting, what i found was with `deep` is it was reflecting the change in the Fee Summary as soon as you edited the field, and by removing it it would update once the form was 'done' editing etc

Reply:
That's weird, since Fee Summary (or its data) isn't part of the businessContact object. Maybe you just saw a timing issue. Either way, I think you need "deep" or this won't recognize changes to businessContact fields.

Reply:
The fee summary is actioned by the Getter that looks at the `changed` prop in the various areas of correction. In this case, changing anything in the `YourCompany` will reflect in the `defineCompanyStep.changed` becoming true. 

Can confirm, with `deep` it reflects in the Fee summary as soon as any edits happen within the text field. Without, the changes in the fee summary reflect when selecting 'done'.

Will look into it more, for a better explanation!

Reply:
So after another look..
Inside the watcher we were creating a direct reference from the model to the prop, which is why `deep: true` was causing the Prop to change whenever the model changed. In turn causing the watcher to call itself often, where we only wanted it to be reactive to the store values, ie `businessContact`. 
So, put `deep: true` back in, but used a shallow copy of the prop to initialize the `contact` data opposed to a reference to the prop itself.

Reply:
Yes after the initial commit it was a direct reference to the prop. But I had fixed this in PR https://github.com/bcgov/business-edit-ui/pull/36/files and fee summary was working correctly for business contact save and resume. Looks like it got overwritten by some other PR

Reply:
Thanks for the fix!